src/main/java/org/mtgpeasant/decks/burn/BurnDeckPilot.java

```diff
@@ -580,10 +601,44 @@ private void sacrificeAMoutain() {
     }
 
     private void scry(int number) {
```

Comment:
I like the overall philosophy
strategy about lands is great
I'm wondering whether the strategy about keeping creatures should be **per creature** or **global** (something like _3 creatures is_ enough for a burn deck?)
really not sure about this, I'm just wondering...

Reply:
maybe you could make some tests with several strategies and keep the best one (the one that optimizes the average kill turn ?)

Reply:
I think the best strategy would be far more complex. 3 thermos is too much for a kill on turn 4 but 3 ghitu lavarunners is not. Moreover, the best strategy against goldfish is not the best strategy in the real world. Buts tests is a good idea, at least for lands.